test/Recollateralization.test.ts

```diff
@@ -3248,7 +3249,6 @@ describe(`Recollateralization - P${IMPLEMENTATION}`, () => {
                 collateral1.address,
                 collateral0.address,
                 issueAmount,
-                config.minTradeVolume,
```

Comment:
kind of worried about this change since the function is not edited in this pr, meaning there might have been something wrong with this test?

Reply:
lol looks like `config.minTradeVolume` was set to `fp('1e-2')` and `config.maxTradeSlippage` set to `fp('0.01')`, so we're all good here.

Reply:
I also confirmed by hand this test was running properly